python/hsfs/core/job_api.py

```diff
@@ -35,3 +35,16 @@ def launch(self, name):
         path_params = ["project", _client._project_id, "jobs", name, "executions"]
 
         _client._send_request("POST", path_params)
```

Comment:
shouldn't this return the job?

Reply:
This actually returns an execution object not a job object.